doc/dev/GOVERNANCE.md

```diff
@@ -0,0 +1,40 @@
+# Governance Policy
+
+This document provides the governance policy for the F3D Project. Maintainers agree to this policy and to abide by all Project polices, including the [code of conduct](../../CODE-OF-CONDUCT.md) by adding their name to the [maintainers.md file](./MAINTAINERS.md).
```

Comment:
I think 'project' should be lowercase, its used throughout the document with both capital P and lowercase p, so at a minimum it should at least be consistent

Reply:
I would keep `Project` with an uppercase, but we need to define it when it appears the first time, just like other specific terms like `Maintainers` or `Contributors`. Something like:

```suggestion
This document provides the governance policy for F3D (the "Project"). [Every person](./MAINTAINERS.md) responsible for its maintenance (the "Maintainers") agrees to this policy and abides by all Project policies, including the [code of conduct](../../CODE-OF-CONDUCT.md) by adding their name to the [maintainers.md file](./MAINTAINERS.md).
```